README.md

```diff
@@ -99,6 +99,7 @@ packages:
 ## Package Maintenance
 The Fivetran team maintaining this package _only_ maintains the latest version of the package. We highly recommend that you stay consistent with the [latest version](https://hub.getdbt.com/fivetran/google_play_source/latest/) of the package and refer to the [CHANGELOG](https://github.com/fivetran/dbt_google_play_source/blob/main/CHANGELOG.md) and release notes for more information on changes across versions.
 
+## Opinionated Decisions
```

Comment:
This package includes a DECISIONLOG.md; however, it is never referenced within the README. I felt this would be a good location to reference the DECISIONLOG.

Would you be able to add that reference here.